functions/src/modules/SE.functions.ts

```diff
@@ -194,94 +194,92 @@ export const createTaskFromChunks = functions.database
     });
     return snapshot.val();
   });
-  return snapshot.val();
-});
 
 
 
 /**
  * Sends a notification email to the coordinator & udpates the corresponding Task
  */
-const seUploadsBucketUrl = functions.config().sound_editing.uploads.bucket_name;
-export const processUploadedFile = functions.storage.bucket(seUploadsBucketUrl).object()
-.onFinalize(async object => {
+const stoargeBaseDomain = functions.config().storage['root-domain'];
+export const processUploadedFile = functions.storage.bucket(`uploads${stoargeBaseDomain}`).object()
+  .onFinalize(async object => {
 
-  const filePath = object.name;
-  const seUploadsBucket = admin.storage().bucket(seUploadsBucketUrl);
-  let uploadCode, list, taskId, task, taskRef;
+    const filePath = object.name;
+    const seUploadsBucket = admin.storage().bucket(`uploads${stoargeBaseDomain}`);
```

Comment:
I guess you can do this to avoid duplicating the bucket name construction.
`admin.storage().bucket(object.bucket);`

Reply:
And name the variable `uploadsBucket`.